prisma/migrations/20241210163200_add_value_agent_identifier/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- CreateEnum
+CREATE TYPE "AgentIdentifierType" AS ENUM ('ORCID', 'ID_REF', 'ID_HAL', 'SCOPUSEID');
```

Comment:
Please keep the syntax consistent with https://github.com/CRISalid-esr/crisalid-ikg/blob/56bf8cb5adbdaae189a4617f8d98c185e90f82e3/app/models/identifier_types.py#L12